test/tstDetailsKokkosExtKernelStdAlgorithms.cpp

```diff
@@ -41,13 +45,10 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(nth_element, DeviceType, ARBORX_DEVICE_TYPES)
   {
     int const n = v_ref.size();
 
-    Kokkos::View<float *, DeviceType> v("v", n);
-    Kokkos::deep_copy(
-        space, v,
-        Kokkos::View<float *, Kokkos::HostSpace,
-                     Kokkos::MemoryTraits<Kokkos::Unmanaged>>(v_ref.data(), n));
+    Kokkos::View<float *, DeviceType> v("Testing::v", n);
+    Kokkos::deep_copy(space, v, UnmanagedView<float>(v_ref.data(), n));
```

Comment:
Why do we care to copy with a specific exec space if we constructed the view without it?
Can't just use create_mirror_view_and_copy() ?